crt_portal/cts_forms/model_variables.py

```diff
@@ -240,10 +240,12 @@
 NEW_STATUS = 'new'
 OPEN_STATUS = 'open'
 CLOSED_STATUS = 'closed'
+REJECTED_STATUS = 'rejected'
 STATUS_CHOICES = (
     (NEW_STATUS, 'New'),
     (OPEN_STATUS, 'Open'),
     (CLOSED_STATUS, 'Closed'),
+    (REJECTED_STATUS, 'Rejected'),
 )
```

Comment:
This is adding a choice to the overall report statuses - is that intentional?

e.g. will it make sense to filter reports by "Rejected" from the /form/view page?


Maybe having an explicit disposition_status field on report would make more sense?